cmd/restic/cmd_check.go

```diff
@@ -479,3 +506,15 @@ func selectRandomPacksByFileSize(allPacks map[restic.ID]int64, subsetSize int64,
 	packs := selectRandomPacksByPercentage(allPacks, subsetPercentage)
 	return packs
 }
+
+type checkResult struct {
+	Error           bool     `json:"error"`
+	NumIndexErrors  int      `json:"num_index_errors"`
+	NumPackErrors   int      `json:"num_pack_errors"`
+	NumTreeErrors   int      `json:"num_tree_errors"`
+	NumUnusedBlobs  int      `json:"num_unused_blobs"`
+	NumDataErrors   int      `json:"num_data_errors"`
```

Comment:
What is a `data` error in contrast to a `pack` error? Both refer to pack files.

Reply:
I unified the error counts per https://github.com/restic/restic/pull/5194#discussion_r1899856559.